### ISSUE TITLE
Enchantments on defined items not working

### ISSUE THREAD

**Issue by MihaiChirculete** [LMBishop/Quests#407]
### Description of Bug

It would seem that defining items with enchantments on them in 1.19 spigot does not work.

Below is the definition of my quest item. Everything else works except the enchantments.

`type: "defined"
item:
  name: "&6&lGreatsword of a Fallen Knight"
  item: GOLDEN_SWORD
  unbreakable: true
  lore: 
    - "&aGreatsword used by a brave Knight in many battles against evil,"
    - "&athat eventually faced their inevitable death."
    - "&aCrafted from ancient magic materials, this blade cannot be shattered."
    - ""
    - "&5The sword carries a powerful energy from it's former master..."
  enchantments:
    - "minecraft:fire_aspect:2"
    - "minecraft:vanishing_curse:1"
  attributemodifiers:
    - attribute: GENERIC_ATTACK_DAMAGE
      modifier:
        uuid: "49dc07dc-bfdb-4dc7-85d3-66ef52b51857"
        name: "generic.attackDamage"
        operation: ADD_NUMBER
        amount: 11
        equipmentslot: HAND`

### Steps to Reproduce

1. Define the above item in an yml file.
2. Reload the plugin and check the item was loaded.
3. Aquire the item in-game.
4. See that the enchantments are missing from it, but everything else works.

### Environment

- Minecraft version: 1.19
- Server software: Spigot
- Quests version: 3.12

### Agreements

- [X] I am running the latest version of Quests
- [X] I am certain this issue is unique and a similar issue is not currently open
- [X] I am using a version of Minecraft which is supported by Quests

**Comment by LMBishop** [comment]
I think the plugin uses the stupid names which Bukkit gave the enchantments since I couldn't figure out the namespace key thing, try with `ARROW_INFINITE` and see if that works

**Comment by MihaiChirculete** [comment]
Yep, that one works. Basically the ones I used work too but it would seem that they must be in capital form.

For example the first one doesn't work but the second does:
```yml
- "minecraft:vanishing_curse:1"
- "minecraft:VANISHING_CURSE:1"
```
